.studiorc

```diff
@@ -0,0 +1,288 @@
+#!/bin/bash
+
+export HAB_DEPOT_CHANNEL
+export APP_HOSTNAME
+export GITHUB_API_URL
+export GITHUB_WEB_URL
+export GITHUB_CLIENT_ID
+export GITHUB_CLIENT_SECRET
+export GITHUB_ADMIN_TEAM
+export GITHUB_WORKER_TEAM
+export WORKER_AUTH_TOKEN
+export SSL_CERT_FILE
+
+function init-datastore {
+  mkdir -p /hab/svc/builder-datastore
+  cp -f /src/support/builder/datastore.toml /hab/svc/builder-datastore/user.toml
+  /src/support/builder/init-datastore.sh
+}
+
+function configure {
+  sup-term
+  sleep 2
+  /src/support/builder/config.sh
+  sleep 2
+  sup-run
+}
+
+function install-packages {
+  hab pkg install core/cacerts core/busybox-static core/hab-launcher core/postgresql -b -c stable
+  hab pkg install \
+    core/builder-admin \
+    core/builder-admin-proxy \
+    core/builder-api \
+    core/builder-api-proxy \
+    core/builder-datastore \
+    core/builder-jobsrv \
+    core/builder-originsrv \
+    core/builder-router \
+    core/builder-scheduler \
+    core/builder-sessionsrv \
+    core/builder-worker -c unstable
+}
+
+function build-habitat {
+  build-hab
+  build-launcher
+  build-sup
+}
+
+function build-hab {
+  build "components/hab"
```

Comment:
Assuming that we're entering the Studio from the root of the project, you should be safe to call `build "/src/components/hab` so that these functions can be run from any directory.